tools/interactive/interactivetool_paraview.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="interactive_tool_paraview" tool_type="interactive" name="Paraview" version="v5.12+galaxy02" profile="24.2">
     <icon src="paraview.png"/>
     <requirements>
-        <container type="docker">quay.io/sunyi000/docker-paraview:v5.12</container>
+        <container type="docker">quay.io/bmcv/docker-paraview:v5.12</container>
```

Comment:
Fix the version of the image:
```suggestion
        <container type="docker">quay.io/bmcv/docker-paraview:v5.11</container>
```
See [this comment](https://github.com/usegalaxy-eu/galaxy/pull/317/files#r2276671088) for details.